homework4/task2.js

```diff
@@ -0,0 +1,8 @@
+function plusFactory(a) {
+    function plusF(b){
+         console.log(a+b) 
+        
+    }
+      plusF(10)
+   }
+   plusFactory(188)
```

Comment:
INCORRECT

Function add 10 every time
Expected result: Should be a possibility to set the b parameter